pkg/scanners/clairify/clairify.go

```diff
@@ -451,7 +453,10 @@ func retryOnGRPCErrors(ctx context.Context, name string, f func() error) error {
 	return backoff.RetryNotify(op, backoff.WithContext(eb, ctx), notify)
 }
 
-func (c *clairify) GetNodeInventoryScan(node *storage.Node, inv *storage.NodeInventory) (*storage.NodeScan, error) {
+func (c *clairify) GetNodeInventoryScan(node *storage.Node, inv *storage.NodeInventory, ir *v4.IndexReport) (*storage.NodeScan, error) {
```

Comment:
Why do we need the 3rd param `ir *v4.IndexReport` here? This code is for handling scanner v2.

Reply:
Ah, I see, it is because of the interface! Then, I think we should rename that function to something not bound to the node-inventory.

Reply:
Planned as [ROX-26475](https://issues.redhat.com/browse/ROX-26475)